### PR TITLE
feat/PRO-2402/top-climbers-vertical-tile

### PR DESCRIPTION
**Pull request by RanaBug**
## Description

- FE implementation of the Tokens vertical tile of PillarX home page

## How Has This Been Tested?

- FE implementation of all components within the tokens vertical tile
- Tests of all new components

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/apps/pillarx-app/components/ClimbersLosersTokens/ClimbersLosersTokens.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            </div>
+        </div>
+    </TileContainer>
+    )
+}
+
+    return (
+    <TileContainer className='px-10 pt-[30px] pb-5 tablet:p-5 mobile:p-0 mobile:bg-[#1F1D23] mobile:flex-col'>
+        <ClimbersLosersList type='climbers' data={topCimbers ?? []} />
+        <ClimbersLosersList type='losers' data={topCimbers ?? []} />
```
Comment (IAmKio) [inline src/apps/pillarx-app/components/ClimbersLosersTokens/ClimbersLosersTokens.tsx:44]: Just a reminder that we'll need to change this soon

File: src/apps/pillarx-app/components/HorizontalToken/HorizontalToken.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                <BodySmall className='text-purple_light mr-[18px]'>0{tokenIndex}</BodySmall>
+                <img src={logo ?? defaultLogo} className='w-[50px] h-[50px] object-fill rounded mr-3.5' />
+                <div className='flex flex-col'>
+                    {tokenSymbol && <Body>{tokenSymbol}</Body>}
+                    {tokenName && <BodySmall className='text-purple_light'>{tokenName}</BodySmall>}
+                </div>
+            </div>
+            <div className='flex flex-col items-center'>
+                {tokenValue && <Body>${tokenValue.toFixed(4)}</Body>}
+                <TokensPercentage percentage={percentage} />
```
Comment (IAmKio) [inline src/apps/pillarx-app/components/HorizontalToken/HorizontalToken.tsx:32]: Can i just check that this doesn't render if there's no `percentage`?

File: src/apps/pillarx-app/components/TrendingTokens/TrendingTokens.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { useTranslation } from 'react-i18next'
 import { createRef, useEffect, useState } from 'react';
 
 // types
-import { TrendingTokenData } from '../../../../types/api'
+import { TokenData } from '../../../../types/api'
```
Comment (IAmKio) [inline src/apps/pillarx-app/components/TrendingTokens/TrendingTokens.tsx:5]: 👍

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few comments but most important is let's tackle the issue with the layout names sooner than later and the title data that should come from the API - but overall, great work!

File: src/apps/pillarx-app/components/HorizontalToken/HorizontalToken.tsx (hunk 2 above)
Comment (RanaBug) [reply src/apps/pillarx-app/components/HorizontalToken/HorizontalToken.tsx:32]: Yes correct, if no percentage the whole component TokensPercentage does not render, the logic is within that component.

File: src/apps/pillarx-app/components/ClimbersLosersList/ClimbersLosersList.tsx
[the remark this answers is not in the document]
Comment (RanaBug) [reply src/apps/pillarx-app/components/ClimbersLosersList/ClimbersLosersList.tsx:1]: Yes absolutely, it has now been renamed

File: src/apps/pillarx-app/components/ClimbersLosersTokens/ClimbersLosersTokens.tsx (hunk 1 above)
Comment (RanaBug) [reply src/apps/pillarx-app/components/ClimbersLosersTokens/ClimbersLosersTokens.tsx]: Yes changed

**Review by IAmKio — APPROVED** [review]
Great work on the rework 🚀

**Pull request merged**